Sources/BraintreeCore/Analytics/BTAnalyticsService.swift

```diff
@@ -48,6 +50,7 @@ class BTAnalyticsService: Equatable {
     /// Exposed to be able to execute this function synchronously in unit tests
     func performEventRequest(
         _ eventName: String,
+        request: Bool? = nil,
```

Comment:
Nitpick - I would call all of these input param names `isVaultRequest`, since `request` is a little vague

Reply:
➕ 1